docs-chef-io/content/inspec/resources/selinux.md

```diff
@@ -79,3 +121,48 @@ The `be_enforcing` matcher tests if the SELinux mode is set to enforcing:
 The `be_permissive` matcher tests if the SELinux mode is set to permissive:
 
     it { should be_permissive }
+
+### be_on
+
+The `be_on` matcher tests if the SELinux boolean is on:
+
+    it { should be_on }
+
+### be_enabled
+
+The `be_enabled` matcher tests if the SElinux module is enabled:
+
+    it { should be_enabled }
+
+## Resource Parameters
+
+- `names`, `status`, `states`, `priorities`,  are valid parameters for SELinux `modules`
+
+- `names`, `status`, `states`, `defaults`,  are valid parameters for SELinux `booleans`
+
+## Resource Parameter Examples
+
+### modules
+
+`modules` returns the information about SELinux modules as returned by [semodule -lfull](https://man7.org/linux/man-pages/man8/semodule.8.html).
+
+Note: The `semodule -l` command does not provide `version` information in newer versions of Linux based systems like RHEL8 and Centos8 so we are not supporting that option [REF](https://access.redhat.com/solutions/2760071).
+
+describe selinux.modules do
+  its("names") { should include "zebra" }
+  its("status") { should include "installed" }
+  its("states") { should include "enabled" }
+  its("priorities") { should include "100" }
+end
+
+### booleans
+
+`booleans` returns the information about SELinux booleans as returned by [semanage boolean -l -n](https://man7.org/linux/man-pages/man8/semanage-boolean.8.html)
+
+describe selinux.booleans do
+  its("names") { should include "httpd_enable_homedirs" }
+  its("states") { should include "on" }
+  its("states") { should include "off" }
+  its("defaults") { should include "on" }
+  its("defaults") { should include "off" }
+end
```

Comment:
````suggestion
```ruby
describe selinux.booleans do
  its("names") { should include "httpd_enable_homedirs" }
  its("states") { should include "on" }
  its("states") { should include "off" }
  its("defaults") { should include "on" }
  its("defaults") { should include "off" }
end
```
````